File: api/src/models/tasks.js
import Sequelize from 'sequelize';
import { sequelize } from '../database/database'
import User from './users';


const Tasks =  sequelize.define('tasks', {
    id:{
        type:Sequelize.INTEGER,
        primaryKey:true,
        autoIncrement: true,
    },
    task_title:{
        type: Sequelize.STRING,
    },
    task_description:{
        type: Sequelize.STRING
    },
    completion_flag:{
        type: Sequelize.STRING
    },
    customer_id:{
        type: Sequelize.INTEGER
    },
   
});

export default Tasks;
User.belongsTo(Tasks, { foreignKey:'id', targetKey: 'customer_id'})
Tasks.hasOne(User, { foreignKey:'id', targetKey: 'customer_id' }) 

File: ReadMail/ReadMail/Models/Tasks.cs
using System;
using System.Collections.Generic;

namespace ReadMail.Models
{
    public partial class Tasks
    {
        public int Id { get; set; }
        public string TaskTitle { get; set; }
        public string TaskDescription { get; set; }
        public string CompletionFlag { get; set; }
        public int? CustomerId { get; set; }
    }
}
